src/visit_vtk/lightweight/vtkEnumThreshold.h

```diff
@@ -135,8 +135,8 @@ class VISIT_VTK_LIGHT_API vtkEnumThreshold : public vtkUnstructuredGridAlgorithm
     bool              allCellsKeptInLastRequestData;
 
   private:
-    vtkEnumThreshold(const vtkEnumThreshold&);  // Not implemented.
-    void operator=(const vtkEnumThreshold&);  // Not implemented.
+    vtkEnumThreshold(const vtkEnumThreshold&) = delete;
```

Comment:
I am noticing a slight variation on this particular pattern.

One way is like this...

```
    vtkEnumThreshold(const vtkEnumThreshold&) = delete;
    void operator=(const vtkEnumThreshold&) = delete;
```

And another looks like this...

```
    vtkEnumThreshold(const vtkEnumThreshold&) {;};
    void operator=(const vtkEnumThreshold&) {;};
```

Don't these basically mean the same thing and if so, is one style perferred over the other?

Reply:
Use of '= delete' is an explicit way of marking the function as deleted. 
I think it is better than the empty '{;}' in that it is very clear that the class is not meant to have these default methods implemented.
Some of the other cases where I made this change only had a comment stating it was not implemented, but no '{;}'.

Reply:
> Some of the other cases where I made this change only had a comment stating it was not implemented, but no '{;}'.

Aren't those cases different from these, in that those are functions we are letting the compiler default initialize, versus explicitly making them do nothing or deleting them?